bambi/plots/effects.py

```diff
@@ -0,0 +1,349 @@
+# pylint: disable = protected-access
+# pylint: disable = too-many-function-args
+# pylint: disable = too-many-nested-blocks
+from dataclasses import dataclass
+import itertools
+from typing import Union, Dict
+
+import arviz as az
+import numpy as np
+import pandas as pd
+import xarray as xr
+
+import bambi as bmb
+from bambi.utils import listify, get_aliased_name
+from bambi.plots.create_data import create_cap_data, create_comparisons_data
+from bambi.plots.utils import identity, Comparison
+
+
+def predictions(
```

Comment:
I like the name of the function. In the future, `plot_cap()` should be called `plot_predictions()` as well.

Reply:
Should I go ahead and make the change here in this branch and PR? Or should a separate PR be made for this? (to avoid too much "other" developments being made in this PR)

Reply:
It can be a separate PR after this is done. No need to worry about it now.